swagger/index.js

```diff
@@ -131,13 +133,21 @@ module.exports = yeoman.Base.extend({
           name: model.name,
           plural: model.plural,
           base: model.base || 'PersistedModel',
-          facetName: 'server', // hard-coded for now
+          facetName: 'common', // hard-coded for now
```

Comment:
It would be great to provide a migration guide. We can review the instructions then to see if there are any opportunities to make the upgrade less cumbersome.